### PR TITLE
feat(ProgressButton): add Link/useSubmit to ProgressButton, close #144

### LINKED ISSUE #144: Add Link to Progress Button
Currently only has href with no ability to Link in React Router

### PR DESCRIPTION
**Pull request by JimBurch**

### CODE REVIEW

File: playground/src/App/index.tsx [hunk 1]
```diff
@@ -22,7 +23,7 @@ import {
   ProgressButton,
   Hero,
   Card,
-  HorizontalRule,
+  // HorizontalRule,
```
Comment (dLars99) [inline playground/src/App/index.tsx:26]: Commented code

File: playground/src/App/index.tsx [hunk 2]
```diff
@@ -143,12 +144,12 @@ const App: React.FC = () => {
 
   return (
     <ThemeProvider>
-      <div style={{ margin: 40 }}>
+      {/* <div style={{ margin: 40 }}>
```
Comment (dLars99) [inline playground/src/App/index.tsx:147]: Commented code

**Review by dLars99 — CHANGES_REQUESTED** [review]
Some commented code to remove. Everything else looks good to me!

File: playground/src/App/index.tsx (hunk 1 above)
Comment (JimBurch) [reply playground/src/App/index.tsx:26]: that's just the playground, yarn link was doin me dirty so I just commented it out but that doesnt affect the package

File: playground/src/App/index.tsx (hunk 2 above)
Comment (JimBurch) [reply playground/src/App/index.tsx:147]: same as above

**Review by wliederer — APPROVED** [review]

**Review by dLars99 — APPROVED** [review]

**Pull request merged**